### DISCUSSION TITLE
Restarting from a checkpoint with a different base grid

### DISCUSSION THREAD

**Discussion by RSuryaNarayan** [AMReX-Combustion/PeleC#342] [Q&A] [answered]
Hi, 

Is there a way to interpolate checkpoint data from a fine base grid (say 1024 x 1024) to a coarse grid (512 x 512)? 

Could I use PeleAnalysis for that in any way? 

Thanks!

**Comment by marchdf** [comment, marked as answer]
Hi, apologies for leaving you hanging here. I completely missed this question. I would recommend using the initialize from plot file capability. This will interpolate a plot file onto any grid and use that as the initial condition (time will be reset to 0): `pelec.init_pltfile = "plt00000"`. More info here: https://amrex-combustion.github.io/PeleC/InputFiles.html

**Reply by RSuryaNarayan** [reply]
Thanks once again! no worries. I will try this out!

**Reply by mikaelm123** [reply]
Hello,
I attempted to start from a plt file and a chk file while changing the grid resolution by doubling it or halving it. Each time I started from a different resolution from either file it throws a Segfault error. However, starting from those files without changing the resolution works just fine. Are there any additional steps to take besides the ones listed above to use the interpolation feature?

Thanks in advance.

**Reply by SKT-Bush** [reply]
Hello mikaelm, I have encountered the same problem, have you solved it?

**Reply by baperry2** [reply]
@SKT-Bush this problem was resolved for restarting on a finer base grid in https://github.com/AMReX-Combustion/PeleC/pull/645. You need to add `pelec.init_pltfile_coarse_levels = 1` to your input file if refining by a factor of two (see documentation: https://amrex-combustion.github.io/PeleC/InputFiles.html#input-files-and-controls).

We do not support restarting on a coarser base grid.

**Reply by SKT-Bush** [reply]
Thank you very much, Mr.baperry2! The documentation is very detailed, I will read it carefully.

**Reply by FernandoIGuerra** [reply]
Dear @marchdf,

I tried this method, reinitialize a case of flame proapgation with obstacles. However, a problem arised whith the EB. All my EBs were set with very high temperatures. How can I solve this matter? 

Since I am considering an adiabatic wall even at the EB, I am not sure if setting eb.eb_siothermal would lead to adiabatic wall condition, no matter the temperature of the eb. I am reinitializing wiht same mesh size.

I was also wondering if adding to PeleC::problem_post_restart() a forcing S_new(i,j,k, UTEMP) = 298 would solve this problem.

Thank you in advance.

<img width="936" height="479" alt="image" src="https://github.com/user-attachments/assets/b0ffcd74-e096-4c8f-b4e0-14770fd02f83" />
